integrations/azure-devops/azure_devops/search_criteria.py

```diff
@@ -0,0 +1,14 @@
+from datetime import datetime, timedelta
+from typing import Any
+
+
+PULL_REQUEST_SEARCH_CRITERIA: list[dict[str, Any]] = [
+    {"searchCriteria.status": "active"},
+    {
+        "searchCriteria.status": "all",
+        "searchCriteria.maxTime": datetime.now() - timedelta(days=30),
+    },
+]
+
+MAX_WORK_ITEMS_PER_QUERY = 400
+WORK_ITEMS_WIQL_QUERY = "select * from workitems"
```

Comment:
couldnt find the usages

Reply:
The 'Work item' in ADO isn't just issues. It can be epics, stories, etc..
Here you can (and probably, should) create the relevant filtering to what's interesting to you to see in Port

Reply:
Lets talk about this face to face